src/main/resources/lang/en_US.json

```diff
@@ -212,7 +211,7 @@
     "guiScaleY": "GUI Y Scale: %scale%",
     "abbreviateSkillXpDenominator": "Abbreviate Skill XP Denominator",
     "otherDefenseStats": "Other Defense Stats (Soul Tether/Cell Alignment)",
-    "hideSpawnPointPlayers": "Hide Spawn Point Players",
+    "hideSpawnPointPlayers": "Hide Players at Hub Spawn Point",
     "farmEventTimer": "Farm Event Timer",
     "dontReplaceRomanNumeralsInItemNames": "Don't Replace in Item Names"
```

Comment:
From the previous PR, what exactly does this mean?

Reply:
It means the game won't render players that stand on the exact block you spawn on when you warp to the hub island.

Reply:
Sorry I meant “ dontReplaceRomanNumeralsInItemNames”

Reply:
This means that the first line (the item's name) of an item's tooltip won't be checked for Roman numerals. It works around an issue where someone had a custom item name like "Boots M" but it got changed to "Boots 1000."

Reply:
Is it a subsetting on a feature?

Reply:
It's a sub setting under replace Roman numerals with numbers I think it's called

Reply:
Alright, I feel like that shouldn't be a setting and just be enabled by default but the string makes sense that way

Reply:
It's disabled by default because some things like enchanted books have numerals in their name.
![image](https://user-images.githubusercontent.com/22475143/146628971-d843e2e5-c2ad-4d48-a419-fcc80230435b.png)